src/Table.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { forwardRef, memo, type ReactNode, type CSSProperties } from "react";
+import { assert } from "tsafe/assert";
+import type { Equals } from "tsafe";
+import { fr } from "./fr";
+import { cx } from "./tools/cx";
+import { symToStr } from "tsafe/symToStr";
+import type { FrClassName } from "./fr/generatedFromCss/classNames";
+
+export type TableProps = {
+    fixed?: boolean;
+    data: ReactNode[][];
+    className?: string;
+    caption?: ReactNode;
+    bordered?: boolean;
+    headers?: ReactNode[];
+    noScroll?: boolean;
+    noCaption?: boolean;
+    style?: CSSProperties;
+    bottomCaption?: boolean;
+    colorVariant?: TableProps.ColorVariant;
+    classes?: Partial<Record<"root", string>>;
+};
+
+export namespace TableProps {
+    type ExtractColorVariant<FrClassName> = FrClassName extends `fr-table--${infer AccentColor}`
+        ? AccentColor
+        : never;
+
+    type NonColorTableClassName =
+        | "no-scroll"
+        | "no-caption"
+        | "caption-bottom"
+        | "layout-fixed"
+        | "bordered";
+
+    export type ColorVariant = Exclude<ExtractColorVariant<FrClassName>, NonColorTableClassName>;
+}
+
+/** @see <https://react-dsfr-components.etalab.studio/?path=/docs/tableau>  */
+export const Table = memo(
+    forwardRef<HTMLDivElement, TableProps>((props, ref) => {
+        const {
+            data,
+            headers,
+            caption,
+            bordered,
+            noScroll,
```

Comment:
```suggestion
            noScroll = false,
```
Every optional props shoud get it's default here.